### ISSUE TITLE
Where have the bootstrap .less files gone?

### ISSUE THREAD

**Issue by robrecord** [Themekraft/_tk#35]
I saw that with version 1.2, the less files have been removed for bootstrap. Can we have them back please?
Thanks!

**Comment by Frederic-C** [comment]
Same problem here, I was used to use _tk with Less files to quicly customize the theme. with pure CSS his is hell as any other basic theme. :/

**Comment by robrecord** [comment]
Thank you Fred, yes I eventually did the same thing.

Should I/we submit a pull request?

**Comment by svenl77** [comment]
Hi @robrecord @Frederic-C 

You are absolutely right. The less files should be included. You are welcome to create a pull request!

Latest Bootstrap pull: https://github.com/Themekraft/_tk/pull/38

Is this the latest version of bootstrap?

**Comment by svenl77** [comment]
Currently v3.3.2

Would be nice to update the theme to the latest version. If you are willing to help, you are more the welcome!

**Comment by Frederic-C** [comment]
Nope, bootstrap just get updated yesterday.
Be careful, when I used the LESS files and compiled I noticed some differences between bootstrap.css generated and the version in _tk. Mainly about webkit prefixes for some CSS rules (webkit rules are present in _tk version and not in bootstrap version) I don't know why these are different rules.

**Comment by svenl77** [comment]
true... all changes in the css should be located here: https://github.com/Themekraft/_tk/blob/master/includes/css/bootstrap-wp.css We could create a bootstrap-wp.less.

We also need to make sure that class names and so on didn't change or adjust it in the theme files.

If someone like to take the job and create a pull we can do an intensive review afterward.

**Comment by robrecord** [comment]
I would like to but don't have time this week; will try to revisit - if someone hasn't done it by then, I'll give it a go.
